codespell_lib/data/dictionary.txt

```diff
@@ -18555,10 +18561,12 @@ provate->private, provide,
 provded->provided
 provder->provider
 provdided->provided
+provdidet->provided, provided, provident,
```

Comment:
You've typed `provided` twice, not `provider` the second time.

I was going to add an issue to catch this in the tests, but they already do:
https://travis-ci.org/github/codespell-project/codespell/jobs/668123853?utm_medium=notification&utm_source=github_status

Reply:
should be fixed in c0b2141